logstash-core/spec/logstash/agent/metrics_spec.rb

```diff
@@ -229,7 +229,7 @@ def mhash(*path_elements)
         # since the pipeline is async, it can actually take some time to have metrics recordings
         # so we try a few times
         try(20) do
-          expect { mhash(:stats, :pipelines, :main, :events) }.not_to raise_error , "Events pipelien stats should exist"
+          expect { mhash(:stats, :pipelines, :main, :events) }.not_to raise_error , "Events pipeline stats should exist"
```

Comment:
I'm not sure about this (it's possible that the message is actually part of the api...)